mscore/importmidi/importmidi_inner.cpp

```diff
@@ -176,7 +176,7 @@ MidiOperations::TimeSigNumerator fractionNumeratorToUserValue(int n)
       else if (n == 21)
             numerator = MidiOperations::TimeSigNumerator::_21;
       else
-            Q_ASSERT_X(false, "Meter::fractionNumeratorToUserValue", "Unknown numerator");
+            {Q_ASSERT_X(false, "Meter::fractionNumeratorToUserValue", "Unknown numerator");}
```

Comment:
note: I do this because compiler will complain when building release builds that -Wempty-body.  Adding these brackets around the Q_ASSERT (which gets removed in release builds) is the way we handled this situation before...